.github/workflows/docker.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: docker
+
+on:
+  # publish on releases, e.g. v2.1.13 (image tagged as "2.1.13" - "v" prefix is removed)
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: macbre/push-to-ghcr@master
```

Comment:
Have to admit I haven't used this one before, but it looked simple enough. It should work with the standard `GITHUB_TOKEN` that github provides. So no need to setup anything.